container_files/public_html/doc/builtin/procedures/Accuracy.md

```diff
@@ -198,6 +198,53 @@ with the same row name as in the input, and the following columns:
 * `score`: predicted value
 * `weight`: the row's assigned weight
 
+### <a name="multilabel"></a>Multi-label mode
+
+The `status` field will contain the recall among top-N performance 
+statistic (e.g. [recall](http://en.wikipedia.org/wiki/Precision_and_recall)) 
+for the classifier, where the labels with the highest score will be chosen as the
+predictions for each example. I.e, each example will register a positive for 
+each of its label(s) that is found among the set of highest scoring labels returned by the classifier.
+The size of this set is determined by the `accuracyOverN` parameter. It follows that a `1.0` recall rate cannot
+be obtained if any example contains more unique labels than the value of the `accuracyOverN`
+parameter.
+
+The `weighted_statistics` represents the average of the 
+per-label statistics.
+
+Here is a sample output, with `accuracyOverN` set to `3`:
+
+```python
+{
+    "status": {
+                "weightedStatistics": {
+                    "recall over top 3": 0.6666666666666666
+                },
+                "labelStatistics": {
```

Comment:
As I specified elsewhere, this should be recallOverTopN: { 1: x, 3: y , 10 z ] with the values corresponding to configuration parameters.